lms/djangoapps/dashboard/sysadmin_urls.py

```diff
@@ -16,4 +16,6 @@
     url(r'^gitlogs/(?P<course_id>.+)$', sysadmin.GitLogs.as_view(),
         name="gitlogs_detail"),
     url(r'^task_queue/?$', sysadmin.TaskQueue.as_view(), name="sysadmin_task_queue"),
+    url(r'^mgmt_commands/?$', sysadmin.MgmtCommands.as_view(), name="sysadmin_mgmt_commands"),
```

Comment:
Good idea, I like the approach of having the same URL with a GET for the list and POST for the execute.